pkg/apis/sensor/v1alpha1/generated.proto

```diff
@@ -596,18 +596,20 @@ message Template {
 }
 
 // TimeFilter describes a window in time.
-// DataFilters out event events that occur outside the time limits.
+// It filters out events that occur outside the time limits.
```

Comment:
Auto-generated file

Reply:
Maybe it's good idea to use `.gitattributes` to mark them as \**generated*\*. It makes reviewers life easier.
https://docs.github.com/en/github/administering-a-repository/customizing-how-changed-files-appear-on-github

E.g.:

```gitattributes
**/generated.proto linguist-generated=true
```

Reply:
👍 , good to know this, would you please come out another PR for it?

Reply:
✅ https://github.com/argoproj/argo-events/pull/739